src/test/java/uk/co/codezen/maven/redlinerpm/rpm/DirectoryScannerFactoryTest.java

```diff
@@ -16,6 +16,6 @@ public void factory()
 
         assertEquals(DirectoryScanner.class, ds.getClass());
         assertEquals(true, ds.isCaseSensitive());
-        assertEquals(false, ds.isFollowSymlinks());
+        assertEquals(true, ds.isFollowSymlinks());
```

Comment:
not sure why. since you handle symlinks manually, you don't want to follow them, do  you?

Reply:
If this option is `false`, the directory scanner does not return symlinks. I think the intent is more about following directory symlinks, but the way the code is structured it seems to conflate the two. I don't know the implications of this if you have directory symlinks, especially if they point outside of the scanned directory tree, but that's not a concern for our usage. The plugin uses the old Ant scanner which long predates the `Path` handling that has real support for symlinks. This is the minimal change I could make to make it work for us.